### PR TITLE
7.2.2 and saving web context

### PR DESCRIPTION
**Pull request by pal-alex**

### CODE REVIEW

File: src/n2o_proto.erl [hunk 1]
```diff
@@ -70,8 +70,8 @@ try_info(Module,M,R,S) ->
 
 pid(Ctx) when is_tuple(Ctx) -> maps:get(pid, element(4, Ctx)).
 save_context(Ctx) ->
-    case ets:whereis(web_context) of
-        undefined -> ets:new(web_context, [public, named_table]); % key = sid
+    case ets:info(table_name) of
```
Comment (5HT) [inline src/n2o_proto.erl:73]: is this atom ?

File: priv/n2o.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ var $io = {}; $io.on = function onio(r, cb) {
         if (debug) console.log(evalex);
         try { eval(evalex); return { status: "ok" }; }
         catch (e) { console.log(evalex);
-            console.error("Eval failed:",e); 
-            return { status: '' }; }
+                    console.error("Eval failed:",e); 
+                    return { status: '' }; }
```
Comment (5HT) [inline priv/n2o.js]: This was already merged previously.

File: src/n2o_proto.erl [hunk 3]
```diff
@@ -45,6 +45,7 @@ init(_Transport, Req, _Opts, _) ->
     put(context,Zero),
     Ctx  = fold(init,Zero#cx.handlers,Zero),
     put(context,Ctx),
+    save_context(Ctx),
```
Comment (5HT) [inline src/n2o_proto.erl:48]: do we need here put and save_context both ?

File: src/n2o_proto.erl (hunk 1 above)
Comment (pal-alex) [reply src/n2o_proto.erl]: There is a mistake. I'll fix it.

File: priv/n2o.js (hunk 2 above)
Comment (pal-alex) [reply priv/n2o.js]: Yes, I tried to fix the indentation.

File: src/n2o_proto.erl (hunk 3 above)
Comment (pal-alex) [reply src/n2o_proto.erl:48]: Yes.

**Pull request merged**